plugins/vst_base/VstPlugin.cpp

```diff
@@ -391,9 +400,9 @@ bool VstPlugin::processMessage( const message & _m )
 #ifdef LMMS_BUILD_WIN32
 			// We're changing the owner, not the parent,
 			// so this is legal despite MSDN's warning
-			SetWindowLongPtr( (HWND)(intptr_t) m_pluginWindowID,
-					GWLP_HWNDPARENT,
-					(LONG_PTR) gui->mainWindow()->winId() );
+			//SetWindowLongPtr( (HWND)(intptr_t) m_pluginWindowID,
```

Comment:
I'm not sure what this is for or why it was removed. @DomClark, you added this code in fcc883f8873974cdb8f3f09e715912a8ec039139. Can you chime in?

Reply:
From my point of view:

It's complicated.

When you do stuff like this, it solves some problems but also creates others. What this does, short version, it links processes together in a single message queue and therefor when one hangs, the other one does too. 

I think this was the first version because something similar happens when the user selects "plugging embedding". I deleted it so nothing happens when they select "no embedding" and normally the same things happen when they select qt or win32 embedding.

Reply:
This is here to make plugin UIs always-on-top when not embedded. It does indeed attach the input queues, but that shouldn't cause any extra problems since the queues are attached in embed mode anyway. Also, plugins are generally designed to have the same input queue as the host, since most hosts load the plugin into the host's own process, so this isn't exactly a bad idea.